conf/spgwu.json

```diff
@@ -8,16 +8,24 @@
     "": "max UE sessions",
     "max_sessions": 50000,
 
-    "": "max IP frag table entries (for IPv4 reassembly)",
-    "max_ip_defrag_flows": 1000,
+    "": "max IP frag table entries (for IPv4 reassembly). Update the line below to `\"max_ip_defrag_flows\": 1000` to enable",
+    "": "max_ip_defrag_flows: 1000",
 
-    "": "Gateway interfaces",    
+    "": "Update the line below to `\"ip_frag_with_eth_mtu\": 1518` to enable",
+    "": "ip_frag_with_eth_mtu: 1518",
+
+    "": "Telemetrics-See this link for details: https://github.com/NetSys/bess/blob/master/bessctl/module_tests/timestamp.py",
+    "measure": false,
+
+    "": "Gateway interfaces",
     "s1u": {
         "ifname": "ens803f2"
     },
 
+    "": "UE IP Natting. Update the line below to `\"ip_masquerade\": \"<ip> [or <ip>]\"` to enable",
     "sgi": {
-        "ifname": "ens803f3"
+        "ifname": "ens803f3",
+	"": "ip_masquerade: 18.0.0.1 or 18.0.0.2 or 18.0.0.3"
```

Comment:
comma separated instead of 'or'

Reply:
I am using this exact string literal directly as a bpf expression for the filter module. This field, as it is, simplifies `spgw.bess` a bit.